pallets/transaction-payment-free/src/lib.rs

```diff
@@ -1,8 +1,9 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 
 use codec::{Decode, Encode};
-use frame_support::weights::{DispatchInfo, PostDispatchInfo};
+use frame_support::dispatch::{DispatchInfo, PostDispatchInfo};
 use scale_info::TypeInfo;
+use sp_runtime::traits::SignedExtensionMetadata;
```

Comment:
linting is sayng `sp_runtime::traits::SignedExtensionMetadata` is unused. Try removing?

Reply:
Yeah, just caught that.